core/services/vrf/v2/coordinator_v2x_interface.go

```diff
@@ -0,0 +1,360 @@
+package v2
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/log"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/vrf_coordinator_v2"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/vrf_coordinator_v2plus"
+	"github.com/smartcontractkit/chainlink/v2/core/services/vrf/vrfcommon"
+)
+
+// CoordinatorV2_X is an interface that allows us to use the same code for
+// both the V2 and V2.5 coordinators.
+type CoordinatorV2_X interface {
```

Comment:
Maybe `CoordinatorV2X` is better as a name?

Reply:
CoordinatorV2Base